README.md

```diff
@@ -47,15 +47,37 @@ Information about the latest release and all releases are below. Please note, yo
 
 ## Cities Using MDS
 
-More than 80 cities and public agencies around the world use MDS, and it has been implemented by most major mobility providers. Below are links to some specific agency programs/policies:
-
-* Los Angeles: The rules and guidelines for the Los Angeles Dockless Bikeshare Systems / Pilot Program can be found on [Council Clerk Connect](https://cityclerk.lacity.org/lacityclerkconnect/index.cfm?fa=ccfi.viewrecord&cfnumber=17-1125) along with supporting info on [ladot.io](https://ladot.io/programs/dockless/).
-* Santa Monica: The rules and guidelines for the Santa Monica Shared Mobility Pilot Program can be found at https://www.smgov.net/sharedmobility.
-* Austin: The rules and guidelines for Austin's Micromobility Program can be found at https://austintexas.gov/micromobility.
```

Comment:
@schnuerle i just found out our URL has changed to `https://austintexas.gov/department/shared-mobility-services`. You mind updating this and making it an anchor tag? e.g., 

> Austin: The rules and guidelines for Austin's Micromobility Program can be found on Austin's [Shared Mobility program](https://austintexas.gov/micromobility) website.

Reply:
Ok I made that update, thanks for the correction!